crates/sal-3dlib-core/src/ops/boolean/volume.rs

```diff
@@ -19,3 +19,26 @@ pub struct VolumeConf {
     /// so all fields of [OpConf] also make sense.
     pub op_conf: OpConf,
 }
+///
+/// Algorithm to volume together three sets of the objects.
```

Comment:
This line is not enough to explain what `AlgoMakerVolume` does.
Lines below are required. But not enough again, because purpose of each of <A, B, C> - are must ti be described also.

It is general remark to most of your doc comments